--- Makefile.PL.orig	2006-10-06 08:44:30.403840967 -0700
+++ Makefile.PL	2006-10-06 08:44:42.180431002 -0700
@@ -5,7 +5,7 @@
 WriteMakefile(
     'NAME'		=> 'Readonly::XS',
     'VERSION_FROM'	=> 'XS.pm', # finds $VERSION
-    'PREREQ_PM'		=> {Readonly => 1.02}, # e.g., Module::Name => 1.1
+    'PREREQ_PM'		=> {}, # e.g., Module::Name => 1.1
     ($] >= 5.005 ?    ## Add these new keywords supported since 5.005
       (AUTHOR     => 'Eric Roode <roode@cpan.org>') : ()),
     'LIBS'		=> [''], # e.g., '-lm'
